sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -104,6 +104,12 @@ class SessionCatalog(
 
   private val validNameFormat = "([\\w_]+)".r
 
+  private val cachedCatalogTable = {
```

Comment:
This is a cache, not a table. So the variable name would make more sense if it were `catalogTableCache`.